understanding/22/accessible-authentication.html

```diff
@@ -40,19 +40,20 @@ <h2>Cognitive function test definition</h2>
     <section id="intent">
         <h2>Intent of Accessible Authentication</h2>
 
-        <p>The purpose of this success criterion is to ensure there is an accessible, easy-to-use, and secure method to log in and access content. Most web sites rely on usernames and passwords for logging in. Memorizing a username and password (or transcribing it manually) places a very high or impossible burden upon people with certain cognitive disabilities. </p>
+        <p>The purpose of this success criterion is to ensure there is an accessible, easy-to-use, and secure method to log in and access content. Most web sites rely on usernames and passwords for logging in. Memorizing a username and password (or transcribing it manually) places a very high or impossible burden upon people with certain cognitive disabilities.</p>
  
         <p>Remembering a password is a <a>cognitive function test</a>. Such tests are known to be problematic for many people with cognitive disabilities. Whether it is remembering random strings of characters, a pattern gesture to perform on a touch screen, or identifying which images include a particular object, cognitive function tests will exclude some people. When a cognitive function test is used, at least one other authentication method must be available which is not a cognitive function test.</p>
             
-        <p class="note">Web sites can employ username (or email) and password inputs as an authentication method if it is properly marked-up and does not block copy-paste functionality. If the login form meets <a href="identify-input-purpose">Success Criterion 1.3.5 Input Purpose</a>,  <a href="info-and-relationships">Success Criterion 1.3.1: Info and Relationships</a>, and does not block copy-paste then browser features or password managers can save the user’s information and refill the login.</p>
```

Comment:
I think it would be better to alter the "does not block" bit, we do need to say that websites can employ username & passwords, we got a ton of comments about that last time.

Reply:
Nevermind, saw it moved further down.